lib/evmone/instructions.hpp

```diff
@@ -515,6 +565,7 @@ inline void blobbasefee(StackTop stack, ExecutionState& state) noexcept
     stack.push(intx::be::load<uint256>(state.get_tx_context().blob_base_fee));
 }
 
+// NOLINTNEXTLINE(bugprone-exception-escape)
```

Comment:
`std::get()` raises this, but it's safe because it follows `std::get_if()` with the opposite option. This is weird, because we have the same pattern in other places (`eof.cpp`) and it's not flagged there.